src/routes/Measurements/components/nettests/Dash.js

```diff
@@ -94,7 +96,7 @@ export class DashDetails extends React.Component {
           <div className='col-xs-6'>
             <button className='btn btn-secondary' onClick={() => this.toggledAdvanced()}>
               <FormattedMessage
-                id='nettests.ndt.More'
+                id='nettests.dash.More'
```

Comment:
You should not change this key as the string `More` is already translated for ndt and there is no need to have people translate `More` in the context of Dash again.